.github/workflows/create-release.yml

```diff
@@ -0,0 +1,61 @@
+name: "Create release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      milestone_title:
+        description: 'Milestone title'
+        required: true
+
+env:
+  MAIN_BRANCH: "master"
+  RELEASE_BRANCH: "release"
+
+jobs:
+  create-changelog-PR:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout spinalcordtoolbox
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.RELEASE_BRANCH }}
+
+    - name: Install spinalcordtoolbox
+      run: |
+        ./install_sct -iy
+
+    - name: Update requirements-freeze.txt
+      run: |
+        source python/etc/profile.d/conda.sh
+        conda activate venv_sct
+        pip freeze | grep -v "-e git+" | grep -v "torch" > requirements-freeze.txt
+        conda deactivate
+        echo "# Platform-specific torch requirements (See SCT Issue #2745)" >> requirements-freeze.txt
+        grep "torch" requirements.txt >> requirements-freeze.txt
+
+    - name: Update version.txt
+      run: |
+        echo "${{ github.event.inputs.milestone_title }}" > spinalcordtoolbox/version.txt
+
+    - name: Commit and push
+      run: |
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --global user.name "GitHub Actions Bot"
+        git fetch
+        git checkout ${{ env.MAIN_BRANCH }}
+        git checkout ${{ env.RELEASE_BRANCH }}
+        git merge --no-ff ${{ env.MAIN_BRANCH }}
+        git add requirements-freeze.txt
+        git commit -m "Update requirements-freeze.txt for ${{ github.event.inputs.milestone_title }}"
+        git add spinalcordtoolbox/version.txt
+        git commit -m "Update version.txt for ${{ github.event.inputs.milestone_title }}"
+        git push
+
+    - uses: ncipollo/release-action@v1
+      name: Create release
+      with:
+        commit: ${{ env.RELEASE_BRANCH }}
+        tag: ${{ github.event.inputs.milestone_title }}
+        token: ${{ secrets.GITHUB_TOKEN }}
+        bodyFile: ".github/workflows/release-body.md"
```

Comment:
Ah, one other thing I've just noticed. Currently this workflow will create a release immediately, but the steps in the Wiki say to "Wait for GitHub Actions to pass on `release`." before creating a new release.

I could try to delay this step, or we could simply trust that the cron job will have picked up any errors ahead of time (since it runs the same platforms as `release` does).

Reply:
My vote is to ultimately get rid of the `release` branch: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3163#issuecomment-759074560.

The reasons we need to wait for tests to pass on it are:

- the PR done by your [automate-changelog-PR](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a0b02cb485b54d89e6942c232b7f20a30e6b8353/.github/workflows/automate-changelog-PR.yml) script runs tests, but only the common-platforms subset of tests; whereas [merging to `release`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a0b02cb485b54d89e6942c232b7f20a30e6b8353/.github/workflows/tests.yml#L8) triggers all the tests
- the `release` branch diverged from `master` a long time ago -- someone did `git rm -r` in a bunch of places, and if we're being exhaustive we should probably test this too

But I would much rather release directly from `master` (much easier once we get `pip` working because it will know what files not to include).

In the meantime, we should change those instructions to read "wait for the changelog's PR to pass on `master`.

Proposal: add to this PR the deletion of this line:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a0b02cb485b54d89e6942c232b7f20a30e6b8353/.github/workflows/tests.yml#L8

to encourage people to only test on `master`.

Reply:
I agree with the comment above, we could remove it in the future.